### PR TITLE
Added inc dec stock feature in user guide

### PR DESCRIPTION
**Pull request by hauchongtang**
Make the Feature more intuitive as well -> supplement with screenshots.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -181,6 +225,31 @@ Find suppliers based on name or item
...
+**Steps**
+
+1. Take a look at the inventory panel at the bottom-middle section of the application.
+2. Decide on which item you want to modify the stock of.
+3. Hover your mouse over item and the white text box with a value `1` in it.
+4. Key in your desired amount to add or decrease by.
+
+![Ui](images/incdeckeyin.png)
+
+5. Press the `+` icon to add your desired amount (or press the `-` icon to minus your desired amount).
```
Comment (dreammac3816547290) [inline docs/UserGuide.md:242]: "Minus" might be a bit informal, can use reduce instead

**Review by dreammac3816547290 — COMMENTED** [review]
Looks good, just a small thing to note with the "minus"

File: docs/UserGuide.md (hunk 1 above)
Comment (hauchongtang) [reply docs/UserGuide.md:242]: add a suggestion and commit it.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -181,6 +225,31 @@ Find suppliers based on name or item
...
+
+![Ui](images/afterinc.png)
+
+***Notice that the current stock of Ginger increased by your desired amount.***
+
+**Limitations**
```
Comment (Kang-Quan) [inline docs/UserGuide.md:248]: Good that restrictions are stated explicitly

File: docs/UserGuide.md [hunk 3]
```diff
@@ -28,6 +30,48 @@ Salesy is a desktop app for helping food vendors manage details of their clients
...
+
+Inventory Status in the [Stats Panel](#Statistics Panel).
+
+Inventory Cards for Items
+
+![InventoryCardRed](images/redstatuscard.png)
+
+![InventoryCardOrange](images/orangestatuscard.png)
+
+![InventoryCardGreen](images/greenstatuscard.png)
```
Comment (Kang-Quan) [inline docs/UserGuide.md:64]: Good use of diagram to simplify explanation.

**Review by Kang-Quan — APPROVED** [review]
LGTM.

**Pull request merged**